src/Razor/src/Microsoft.VisualStudio.DevKit.Razor/Microsoft.VisualStudio.DevKit.Razor.csproj

```diff
@@ -1,40 +1,34 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(DefaultNetCoreTargetFrameworks)</TargetFrameworks>
-    <PublishTargetFramework>net8.0</PublishTargetFramework>
+    <OutputType>Library</OutputType>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
```

Comment:
Nit: Could you keep `<TargetFramework>` at the top to be consistent with our other projects? Also, isn't `<OutputType>Library</OutputType>` the default?

Reply:
It is, but I didn't know that off the top of my head and put it there to be explicit. Will fix ordering